src/main/java/com/fryrank/dal/ReviewDALImpl.java

```diff
@@ -32,22 +29,36 @@ public class ReviewDALImpl implements ReviewDAL {
 
     @Override
     public GetAllReviewsOutput getAllReviewsByRestaurantId(@NonNull final String restaurantId) {
-        final Query query = new Query();
         final Criteria equalToRestaurantIdCriteria = Criteria.where(RESTAURANT_ID_KEY).is(restaurantId);
-        query.addCriteria(equalToRestaurantIdCriteria);
-        final List<Review> reviews = mongoTemplate.find(query, Review.class);
-
-        return new GetAllReviewsOutput(reviews);
+        final MatchOperation filterToRestaurantId = match(equalToRestaurantIdCriteria);
+
+        LookupOperation lookup = LookupOperation.newLookup()
+                .from("user-metadata")
+                .localField("accountId")
+                .foreignField("_id")
+                .as("userMetadata");
```

Comment:
Let's convert these "magic strings" into constants.

Reply:
Done in [98ac0c5](https://github.com/NickPriv/FryRankBackend/pull/83/commits/98ac0c5d7f9c27e6e75a242c815fd2e47c9cbdb6).